Core/GDCore/Extensions/PlatformExtension.h

```diff
@@ -5,8 +5,7 @@
  * project is released under the MIT License.
  */
 
-#ifndef GDCORE_PLATFORMEXTENSION_H
-#define GDCORE_PLATFORMEXTENSION_H
+#pragma once

```

Comment:
what does this do?

Reply:
pragma once is the equivalent of ifndef/define/endif

This avoids .h include files to be included more than once. If your .cpp file includes two .h files, and these two .h files include the same one, it must be included only once (otherwise C++ will complain about double declarations of classes/functions).
The answer with recent compilers since a few years is to use #pragma once to ask the compiler to do this deduplication.

Old C++ stuff basically!